vcxagency-node/config/localhost-ca.env

```diff
@@ -13,17 +13,10 @@ AGENCY_WALLET_KEY_SECRET=01234567890123456789
 AGENCY_TYPE=client
 REDIS_URL=redis://localhost:6379/0
 
-PG_STORE_HOST=localhost
-PG_STORE_PORT=5432
-PG_STORE_ACCOUNT=postgres
-PG_STORE_PASSWORD_SECRET=mysecretpassword
-PG_STORE_DATABASE=agency-storage
-
-PG_WALLET_ACCOUNT=postgres
-PG_WALLET_PASSWORD_SECRET=mysecretpassword
-PG_WALLET_ADMIN_ACCOUNT=postgres
-PG_WALLET_ADMIN_PASSWORD_SECRET=mysecretpassword
-
-PG_WALLET_URL=localhost:5432
-PG_WALLET_MAX_CONNECTIONS=90
-PG_WALLET_CONNECTION_TIMEOUT_MINS=30
+MYSQL_HOST=localhost
+MYSQL_PORT=3306
+MYSQL_ACCOUNT=root
+MYSQL_PASSWORD_SECRET=mysecretpassword
+MYSQL_DATABASE_APPLICATION=agency_application
+MYSQL_DATABASE_WALLET=agency_wallets
```

Comment:
Names of the databases is wildly inconsistent across the PR. Sometimes a plural is used, sometimes singular, sometimes both, almost randomly.

Reply:
I can't find any occurrence of "agency_wallet", can you point me to any?

The only deviation I found was that in CI I use "ca_agency_wallets" and "ea_agency_wallets" - but that's because 2 agencies use the same DB server and they should use different schema.

Reply:
Can you use either`agency_wallets` or `wallets_agency`, please?